src/WarriorRogueMage/CharacterCreation/SetAttributesState.cs

```diff
@@ -157,22 +157,22 @@ private void SetAttributeTarget(ref int targetPoints, int newValue)
         private void RefreshScreen(bool sendPrompt = true)
         {
             var sb = new StringBuilder();
-            sb.AppendAnsiLine();
-            sb.AppendAnsiLine();
-            sb.AppendAnsiLine("You have 10 character points to be divided between 3 attributes.");
-            sb.AppendAnsiLine("No attribute can have more than 6 points. Attributes can be zero.");
-            sb.AppendAnsiLine();
-            sb.AppendAnsiLine($"Warrior : {warriorPoints}" + AnsiSequences.NewLine);
-            sb.AppendAnsiLine($"Rogue   : {roguePoints}" + AnsiSequences.NewLine);
-            sb.AppendAnsiLine($"Mage    : {magePoints}" + AnsiSequences.NewLine);
-            sb.AppendAnsiLine();
-            sb.AppendAnsiLine($"You have {MaxPoints - SpentPoints} character points left.");
-            sb.AppendAnsiLine();
-            sb.AppendAnsiLine("<%yellow%>====================================================================");
-            sb.AppendAnsiLine("To add points to an attribute, use the + operator. Example: warrior +6");
-            sb.AppendAnsiLine("To subtract points from an attribute, use the - operator. Example: warrior -6");
-            sb.AppendAnsiLine("When you are done distributing the character points, type done.");
-            sb.AppendAnsiLine("====================================================================<%n%>");
+            sb.AppendLine();
```

Comment:
Odd regression showing up here.
Also with so many commits showing up in the commit list on each PR, I wonder if your fork needs a fresh merge from the base repo master?